images/nginx/1.18/Dockerfile

```diff
@@ -14,9 +14,11 @@ RUN apk add --no-cache \
 RUN mkdir /etc/nginx/certs \
   && echo -e "\n\n\n\n\n\n\n" | openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/nginx/certs/nginx.key -out /etc/nginx/certs/nginx.crt
 
+ARG TARGETARCH
+ENV TARGETARCH=${TARGETARCH:-amd64}
```

Comment:
This line will be removed if we're building using `docker buildx` for the platforms `linux/arm64` and `linux/amd64`

Reply:
I'm gonna go ahead and remove this line for consistency and the better approach.